javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/ConfiguredResourceAttributesHolder.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+public final class ConfiguredResourceAttributesHolder {
+
+  private static final Map<String, String> resourceAttribute = new HashMap<>();
+
+  public static Map<String, String> getResourceAttribute() {
+    return resourceAttribute;
+  }
+
+  public static void initialize(Attributes resourceAttribute) {
+    String[] mdcResourceAttributes = getConfiguredAttributes();
+
+    for (String key : mdcResourceAttributes) {
+      String value = resourceAttribute.get(stringKey(key));
+      if (value != null) {
+        ConfiguredResourceAttributesHolder.resourceAttribute.put(
+            String.format("otel.resource.%s", key), value);
```

Comment:
It looks to me like prepending the `otel.resource` string to the keys is an internal implementation detail. Because it's internal to this class, can we just skip that and do straightforward put/get from the map instead?